[20:59] <590e05cdd73408ce4f5d769b> hi all, i'm trying to run n_jobs>1 with a OneVsRestClassifier(LinearSVC()) and am seeing the following... is this due to lambda's creeping in somewhere recently? ``` Traceback (most recent call last):   File "main.py", line 24, in <module>     train(df, Y)   File "/home/ubuntu/model.py", line 90, in train     mod.fit(docs_train, labels_train) #mod to use grid search or "model" for the pipeline only   File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/sklearn/multiclass.py", line 216, in fit     for i, column in enumerate(columns))   File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/sklearn/externals/joblib/parallel.py", line 768, in __call__     self.retrieve()   File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/sklearn/externals/joblib/parallel.py", line 719, in retrieve     raise exception   File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/sklearn/externals/joblib/parallel.py", line 682, in retrieve     self._output.extend(job.get(timeout=self.timeout))   File "/home/ubuntu/anaconda3/lib/python3.6/multiprocessing/pool.py", line 608, in get     raise self._value   File "/home/ubuntu/anaconda3/lib/python3.6/multiprocessing/pool.py", line 385, in _handle_tasks     put(task)   File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/sklearn/externals/joblib/pool.py", line 371, in send     CustomizablePickler(buffer, self._reducers).dump(obj) AttributeError: Can't pickle local object 'train.<locals>.<lambda>' ```
[21:00] <590e05cdd73408ce4f5d769b> is there a way to set dill or some other serializer instead?
[21:06] <590e05cdd73408ce4f5d769b> nm, found a lambda hidden away in my code somewhere... :)
